Codeinwp/neve
# Allow hosting google fonts locally

### Summary
Added the mechanism for allowing to host google fonts.

This only works if you enable the mechanism via `neve_load_remote_fonts_locally` filter.
It checks if it's not admin or customize preview.
If the filter is enabled and you're meeting the conditions above, the font should be loaded as an inline style.

### Will affect visual aspect of the product
NO


### Test instructions
- In a child-theme add this line of code:
`add_filter( 'neve_load_remote_fonts_locally', '__return_true');` - it suppose to enable the mechanism for loading fonts locally
- In the customizer, make sure you set a Google Font and not a system one. Publish the changes and go on a page where you can see the new font
- Check if the font appears correctly, as it does in the customizer.
- Open Inspect Element and search for 'neve-google-font-'. You should not find any result for this
- If both of the above points are true, it means that the font is loaded locally.


Closes Codeinwp/neve-pro-addon#1107.
